src/nvim/popupmenu.c

```diff
@@ -53,6 +53,7 @@ static int pum_kind_width;          // width of pum items kind column
 static int pum_extra_width;         // width of extra stuff
 static int pum_scrollbar;           // one when scrollbar present, else zero
 static bool pum_rl;                 // true when popupmenu is drawn 'rightleft'
+static char *pum_match;             // characters of match
```

Comment:
I think this needs to be cleared when pum is undisplayed

Reply:
yep. but I am trying to use `VirtText` to refactor pmenu now. so  this patch closed.  use new implement instead.